pkg/logql/syntax/ast.go

```diff
@@ -894,6 +903,19 @@ func (r *LogRange) Walk(f WalkFn) {
 	r.Left.Walk(f)
 }
 
+// WithoutUnwrap returns a copy of the log range without the unwrap statement.
+func (r *LogRange) WithoutUnwrap() (*LogRange, error) {
+	left, err := Clone(r.Left)
```

Comment:
Will be more extensible if we `Clone(self)` then set the unwrap to nil. This ensures that any future fields are propagated through this function.